spaces/domains.json

```
@@ -19,6 +19,7 @@
   "vote.totem.earth": "totemearth.eth",
   "vote.partyhat.finance": "phats.eth",
   "dao.littleghosts.com": "littleghosts.eth",
+  "vote.panicswap.com":"panicswap.eth",
```

Please add a space between sub-domain and ENS.
```suggestion
  "vote.panicswap.com": "panicswap.eth",
```